python_modules/libraries/dagster-airbyte/dagster_airbyte/translator.py

```diff
@@ -11,6 +12,27 @@
 from dagster_airbyte.utils import generate_table_schema, get_airbyte_connection_table_name
 
 
+class AirbyteJobStatusType(str, Enum):
+    RUNNING = "running"
+    SUCCEEDED = "succeeded"
+    CANCELLED = "cancelled"
+    PENDING = "pending"
+    FAILED = "failed"
+    ERROR = "error"
+    INCOMPLETE = "incomplete"
+
+
+@deprecated(breaking_version="1.10", additional_warn_text="Use `AirbyteJobStatusType` instead.")
+class AirbyteState:
```

Comment:
This enum is renamed to be AirbyteJobStatusType, which reflects Airbyte's ontology and our naming convention in other integrations.

Reply:
Is 1.10 the right thing to say here; since Airbyte is on a pre 1.0 release ?